salt/metalk8s/orchestrate/bootstrap/init.sls

```diff
@@ -90,6 +90,16 @@ Bring bootstrap minion to highstate:
     - salt: Sync bootstrap minion
     - salt: Deploy CA role on bootstrap minion
 
+Generate etcd client certs for salt master:
```

Comment:
Is there a reason to have this here, and not have `metalk8s.salt.master.certs` included in `metalk8s.salt.master`, which is deployed on the `bootstrap` role minion?

Reply:
In fact, when the state `metalk8.salt.master` is called, the CA is not deployed yet, so we can't generate the certificate at this moment